src/migrations/1638441120817-canvas-checkout.ts

```diff
@@ -0,0 +1,120 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { RandomGenerator } from 'typeorm/util/RandomGenerator';
+
+export class canvasCheckout21638441120817 implements MigrationInterface {
+  name = 'canvasCheckout21638441120817';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE \`canvas_checkout\` (\`id\` char(36) NOT NULL, \`createdDate\` datetime(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6), \`updatedDate\` datetime(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6), \`version\` int NOT NULL, \`canvasID\` varchar(36) NOT NULL, \`lockedBy\` varchar(36) NOT NULL, \`status\` varchar(255) NOT NULL DEFAULT 'available', \`authorizationId\` char(36) NULL, \`lifecycleId\` char(36) NULL, UNIQUE INDEX \`REL_353b042af56f01ce222f08abf4\` (\`authorizationId\`), UNIQUE INDEX \`REL_bd3c7c6c2dbc2a8daf4b1500a6\` (\`lifecycleId\`), PRIMARY KEY (\`id\`)) ENGINE=InnoDB`
```

Comment:
Does the status need to be varchar(255)? It's a bit excessive.

Reply:
Actually I think it probably should just be 'text' so that it is variable?